mex/editor/login/state.py

```diff
@@ -31,18 +35,32 @@ def set_password(self, password: str) -> None:
     @rx.event
     def login(self) -> Generator[EventSpec, None, None]:
         """Login a user."""
-        write_access = has_write_access_ldap(self.username, self.password)
-        if write_access:
-            encoded = b64encode(f"{self.username}:{self.password}".encode("ascii"))
+        settings = BaseSettings.get()
+        url = urljoin(
+            str(settings.backend_api_url),
+            f"{BackendApiConnector.API_VERSION}/merged-person-from-login",
+        )  # stopgap: MX-2083
+
+        try:
+            response = requests.post(
+                url, auth=(self.username, self.password), timeout=10
+            )
+        except RequestException:
+            yield rx.window_alert("Cannot reach backend. Please try again later.")
```

Comment:
could you use `escalate_error` here? this will give us a more unified error logging